src/components/leader-table.tsx

```diff
@@ -0,0 +1,395 @@
+import * as React from 'react';
```

Comment:
nit: Лучше импортировать из реакта необходимые функции, а не все сразу. Код будет удобнее читать - по импорту видно, что именно используется 

```suggestion
import React, {useState} from 'react';
```